### ISSUE TITLE
[Item] Flow of Knowledge procs on HoT

### ISSUE THREAD

**Issue by AlexCSCP** [dalaranwow/dalaran-wow#5451]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Flow of Knowledge's "on cast" effect procs on HoT ticks.

**Expected behaviour**: Tell us what should happen instead
The description is clear: "Your spell casts have a chance". keyword "casts".
So, it should be triggered by casting a spell.

**Steps to reproduce the problem**:

1. Get the item and equip it.
2. Cast a HoT spell.
3. Wait until one of the ticks procs the buff.

**Include proofs for this behaviour**

http://www.wowhead.com/item=44912/flow-of-knowledge
A comment of someone testing it and not getting procs on HoT ticks.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
44912
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
https://imgur.com/YQKUnke
In the screenshot you can see I got the buff twice from riptides' HoT ticks.

**Comment by Rushor** [comment]
@AlexCSCP: Could you please copy the comment that says, that it should not procc on Hots?

**Comment by AlexCSCP** [comment]
@Rushor
someone posted on wowhead:

> Comment by ZtriDer
> Tested it as resto shaman.
> I did 100+ of each ability to test them out to se if they proc.
> 
> Casting Ghost Wolf – Yes
> Casting weapon enchant (Earthliving/Flametongue/Windfury/frostbrand) - No
> Casting Water walk (Glyphed for free cast) - Yes
> Casting Earth Shield – No
> Earth Shield tick – No
> Casting Water shield – No
> Water Shield Tick – No
> Casting Lighting shield – Yes
> Lighting Shield Tick – Yes O.o
> Casting any totem – No
> Mana Spring Totem Tick - No
> Magma Totem Tick – No
> Fire Nova Totem Explosion - No
> Casting Riptide – Yes.
> Ripetide Heal Tick – No
> Casting Lesser Healing wave - Yes
> Casting Healing Wave – Yes
> Casting Chain Heal – Yes
> 
> Lighting bolt – Yes
> Chain lighting – Yes
> Casting Shocks – Yes
> Flame Shock DoT Tick – No
> 
> A last conclution:
> As for a healing trinket, it my be nice, but not good.
> The hidden CD is to long and the proc chance is way too small compared to the short (10 sec) duration it has.
